components/Button/Button.module.scss

```diff
@@ -32,7 +32,42 @@
     );
 }
 
-.chevron {
+.chevronButton {
+    cursor: pointer;
```

Comment:
**Question**: Is this attribute needed?

Reply:
surprisingly, yes - tested on Chrome using a mac, the button won't show as a pointer by default